### DISCUSSION TITLE
Grid example doesn't work

### DISCUSSION THREAD

**Discussion by rwb196884** [vikramlearning/blazorbootstrap#906] [Q&A]
How do you get the grid to work?

I coped the (client side) example from https://demos.blazorbootstrap.com/grid into my code and nothing appears on the page.

Example in new Blazor project created in VisualStudio2022:
[BlazorApp1.zip](https://github.com/user-attachments/files/17397818/BlazorApp1.zip)

**Comment by gvreddy04** [comment]
@rwb196884 Thank you for trying BlazorBootstrap!

We assume you followed one of our setup guides to integrate BlazorBootstrap. If not, please refer to the documentation: https://docs.blazorbootstrap.com/getting-started/blazor-webapp-server-global-net-8.

Next, try adding a Grid component example.

If you encounter any issues, please share a minimal reproducible example on a public GitHub repository. We cannot accept zip files due to security concerns.


Please share a GitHub repo.

**Comment by ymoona** [comment]
I just followed this getting started:
File new project in visual studio, web assembly standalone app. 
Then: https://docs.blazorbootstrap.com/getting-started/blazor-webassembly-net-8

Then I updated Home.razor (default page) to, see attached file. 
Works like a charm. 
[TestBootstrapGrid.zip](https://github.com/user-attachments/files/17431000/TestBootstrapGrid.zip)

**Reply by rwb196884** [reply]
> I just followed this getting started: File new project in visual studio, web assembly standalone app. Then: https://docs.blazorbootstrap.com/getting-started/blazor-webassembly-net-8

How did you create that project?
When I create a new project I don't get a `wwwroot/index.html` file. In fact I get two projects -- the main server side one and the client side one.

**Comment by rwb196884** [comment]
https://github.com/rwb196884/blazor-bootstrap

It appears that the grid is client-side only; the server side grid does not work.

I cannot follow the instructions at https://docs.blazorbootstrap.com/getting-started/blazor-webapp-server-global-net-8 because they are incorrect. A new Blazor project crated with VisualStudio does not have a file `wwwroot/index.html`. But it seems that you can add the script and CSS to `Components/App.razor` instead.

**Reply by gvreddy04** [reply]
@rwb196884 Checking your branch. I'll get back to you.

**Reply by gvreddy04** [reply]
@rwb196884 Please review the following screenshot and confirm that the same options were selected when creating the project.

![image](https://github.com/user-attachments/assets/6ba8c9d8-0346-468f-a3da-01ed22533e8e)

**Reply by gvreddy04** [reply]
Additional reference: https://github.com/dotnet/aspnetcore/issues/58944#issuecomment-2479063422
